### PR TITLE
List structure

### PR DESCRIPTION
**Pull request by LOctavio**
I built this project using webpack and is served by a webpack dev server.
In this page a user can add tasks and see a list of all of them.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,36 @@
...
+
+- HTML
+- CSS
+- ES6
+- Webpack
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
```
Comment (mricanho) [inline README.md]: - [x] Add instructions about how to run your project locally (e.g. npm install, npm start, etc)

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  const text = document.createElement('label');
+  const icon = new Image();
+  icon.src = Icon;
+  check.type = 'checkbox';
+  text.textContent = description;
+  task.appendChild(check);
+  task.appendChild(text);
+  task.appendChild(icon);
+  container.appendChild(task);
+};
```
Comment (mricanho) [inline src/index.js:30]: - [x] Populate your taskList with examples. And remember to use the 3 dots icon.

Your app:

imagen

Target app:

imagen

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @LOctavio, 

Good job so far! You have great commit messages, and your JS code it's mostly complete, your app's styling it's excellent 🔥🔥

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] Remember that **all your linters must pass** in order to approve this milestone

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

yougotit

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 2 above)
Comment (DeliceLydia) [reply src/index.js:30]: Well addressed this!

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

### Hi @LOctavio, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional
Add in the README each and every step to get app up and running locally like

- Clone the project..
- Cd in the ..
- run `npm install`
- run `npm run build` etc


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**